build.gradle

```diff
@@ -1,7 +1,21 @@
+buildscript {
+	ext {
+		springBootVersion = '1.5.2.RELEASE'
+	}
+	repositories {
+		mavenCentral()
```

Comment:
is this needed?

Reply:
yes, without it, build.gradle give a error.